test/src/test/java/org/corfudb/integration/StreamingIT.java

```diff
@@ -142,8 +144,7 @@ public void testStreamingSingleTable() throws Exception {
 
         // Subscribe to streaming updates from the table.
         StreamListenerImpl s1n1t1 = new StreamListenerImpl("s1n1t1");
-        store.subscribe(s1n1t1, "n1",
-                Collections.singletonList(new TableSchema("t1", Uuid.class, Uuid.class, Uuid.class)), ts1);
+        store.subscribe(s1n1t1, "n1", defaultStreamTag, Collections.singletonList("t1"), ts1);
```

Comment:
Can one of these tests still carry the old subscribe() api since some verticals may still need to use the old form without actually depending on the tables being opened?

Reply:
I expanded the last testcase to use both old and new subscribe api.